conformance/utils/suite/experimental_suite.go

```diff
@@ -138,10 +138,18 @@ func NewExperimentalConformanceTestSuite(s ExperimentalConformanceOptions) (*Exp
 					}
 					suite.extendedUnsupportedFeatures[conformanceProfileName].Insert(f)
 				}
+				// Add Exempt Features into unsupported features list
+				if s.ExemptFeatures.Has(f) {
+					suite.extendedUnsupportedFeatures[conformanceProfileName].Insert(f)
+				}
```

Comment:
@arkodg can you add a follow up issue to prevent core features from being specified as "Exempt"?

Reply:
raised https://github.com/kubernetes-sigs/gateway-api/issues/2524